### PR TITLE
[WIP] Testcase to ensure master node reboot

### PR DESCRIPTION
**Pull request by xenolinux**
Adding a testcase to ensure if the components are healthy after master
node reboot.

Signed-off-by: Servesha Dudhgaonkar <sdudhgao@redhat.com>

### CODE REVIEW

File: tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py [hunk 1]
```diff
@@ -100,7 +101,7 @@ def finalizer():
             if scheduling_disabled_nodes:
                 schedule_nodes(scheduling_disabled_nodes)
 
-            # Validate all nodes are in READY state
+                # Validate all nodes are in READY state
```
Comment (anubhav-here) [inline tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py:104]: Extra spaces for no reason.

Comment (xenolinux) [reply tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py:104]: yes. Will eliminate it.

File: tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py [hunk 2]
```diff
@@ -352,13 +353,27 @@ def test_monitoring_after_rebooting_master_node(self, nodes, pods):
...
+            # Enter in the node and execute reboot
+            for node in master_nodes:
+                utils.run_cmd(f"oc debug node/{node}")
+                utils.run_cmd("reboot")
+
+                wait_for_nodes_status_and_prometheus_health_check(pods)
+        else:
+            # Reboot one after one master nodes
+            for node in master_nodes:
+                nodes.restart_nodes([node])
```
Comment (vasukulkarni) [inline tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py:374]: please add keyword for arg as I am changing it in different pr.  `nodes.restart_nodes(nodes=[node])`

Comment (xenolinux) [reply tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py:374]: ack

File: tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py [hunk 3]
```diff
@@ -352,13 +353,27 @@ def test_monitoring_after_rebooting_master_node(self, nodes, pods):
...
+                utils.run_cmd(f"oc debug node/{node}")
+                utils.run_cmd("reboot")
+
+                wait_for_nodes_status_and_prometheus_health_check(pods)
+        else:
+            # Reboot one after one master nodes
+            for node in master_nodes:
+                nodes.restart_nodes(nodes=[node])
+
+                wait_for_nodes_status_and_prometheus_health_check(pods)
```
Comment (ebenahar) [inline tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py:376]: I suggest we wait for https://github.com/red-hat-storage/ocs-ci/pull/2006 to be merged, which will allow us running restart using IPMI, and this change will not be needed as nodes.restart_nodes() will accommodate this

Comment (xenolinux) [reply tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py:376]: ack!

**Pull request closed without merging**